guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -10,14 +10,14 @@ Modify default strings in your custom {% glossarytooltip d2093e4a-2b71-48a3-99b7
 
 ## How Magento applies locales
 
-When the {% glossarytooltip 05099dbb-d491-4e33-a065-16035cb2d4d9 %}locale{% endglossarytooltip %} is changed for a store, Magento searches for translations in the corresponding dictionaries in the following locations:
+When the {% glossarytooltip 05099dbb-d491-4e33-a065-16035cb2d4d9 %}locale{% endglossarytooltip %} is changed for a store, Magento searches for translations in the corresponding dictionaries are applied in following manner:
 
 1. Module translations: `<module_dir>/i18n/`
 1. Theme translations: 
 	1. `<parent_theme_dir>/i18n/` (iterated through all ancestor themes)
 	1. `<current_theme_dir>/i18n/`
 1. Translation package: `app/i18n/`
-1. Magento database
+1. Magento database (It is good for overriding other translations whereever necessary)
```

Comment:
Can you elaborate on when translations in the Magento database would override other translations? Is it that if translations are present in the Magento database they automatically override other translations for the same string?

Reply:
Hello, @erikmarr 
Yes, database translations are work on top priority,  It is a quick way for store administrators to update and translate text on the Storefront as an inline translation. 

So administrators do not need to modify CSVs. some it may happen, administrators are the non-technical user. It is a Magento good feature.

Thank you !!!